src/os/bluestore/BlueStore.cc

```diff
@@ -5619,6 +5619,25 @@ int BlueStore::allocate_bluefs_freespace(
   return 0;
 }
 
+size_t BlueStore::available_freespace(uint64_t alloc_size) {
```

Comment:
is there any locking required here?

Reply:
Locking is done in method `dump` of Allocator.